media/editor/findWidget.tsx

```diff
@@ -88,7 +88,11 @@ const getSearchQueryOrError = (
 	isBinaryMode: boolean,
 	isRegexp: boolean,
 ): SearchRequestMessage["query"] | string => {
+	const hexPattern = /^\s*([0-9a-fA-F]{2}\s*)+$/;
 	if (isBinaryMode) {
+		if (hexPattern.test(query)) {
+			query = query.replace(/\s/g, "");
+		}
```

Comment:
I think we should just modify `parseHexStringWithPlaceholders` to ignore whitespace in its loop

Reply:
just need add `\?` in hexPattern, like this `/^\s*([0-9a-fA-F\?]{2}\s*)+$/`

Reply:
added by [1083be1](https://github.com/microsoft/vscode-hexeditor/pull/540/commits/1083be1834805b6807298370d35ad526c41814e8)